packages/components/src/panel/row.js

```diff
@@ -8,13 +8,13 @@ import classnames from 'classnames';
  */
 import { forwardRef } from '@wordpress/element';
 
-const PanelRow = forwardRef( ( { className, children }, ref ) => (
+const PanelRow = ( { className, children }, ref ) => (
```

Comment:
Tweaked the export so the displayName isn't borked.

Reply:
Interesting that this is enough! I though we had to name-export `PanelRow`

Reply:
This was super weird, because the original code would show the display name as `[object Object]` 😕  I don't think I've ever seen that before.

Aside from that, I'm still unclear on why behavior seems inconsistent between "named but unexported" and "named and exported". Sometimes it works without an export, sometimes it doesn't 🤷